### DISCUSSION TITLE
how to style AppBody in function?

### DISCUSSION THREAD

**Discussion by SabSchek** [neuefische/questions#172] [Web] [answered]
Hello, 

I've got a question regarding styling the body of the app.

Please see attached the screenshot of my function so far. I can't find the solution to style the div which contains the body of the app. 
If there is someone who's able to help me finding the problem and a way to style the app, it'd be amazing. 

Thanks in advance, Fine

Code:

<img width="838" alt="Bildschirmfoto 2022-09-26 um 18 13 33" src="https://user-images.githubusercontent.com/109827469/192328706-648918f2-6f86-45c4-a06f-e813d59aedb8.png">

Net:
<img width="504" alt="Bildschirmfoto 2022-09-26 um 18 20 01" src="https://user-images.githubusercontent.com/109827469/192329061-e4b9afd6-4736-49da-885e-d32fc8e5bf25.png">

**Comment by pixelass** [comment, marked as answer]
The component `AppBody` has styling (`border-radius`), but you cannot see it since the `AppBody` has no background. 
Try adding `background: red; padding: 10px;` to see that styles are applied.
I'm not sure what your goal is and which styles are applied to the other components.

Please post a link to your project (branch). 


<img width="219" alt="Screen Shot 2022-09-26 at 19 58 39" src="https://user-images.githubusercontent.com/1148334/192347424-d2742add-e562-4030-a8bb-8712d520e126.png">

**Reply by SabSchek** [reply]
It worked, thank you!
